packages/config/config/devices/0x0234/zba7140.json

```diff
@@ -0,0 +1,246 @@
+			"label": "Pushbutton 3 Single-Activation ON Value"
+		},
+		"14[0xff0000]": {
+			"$import": "templates/logic_group_template.json#multilevel_lower_function",
+			"label": "Pushbutton 3 Single-Activation OFF Value"
+		},
+		"14[0xff000000]": {
+			"$import": "templates/logic_group_template.json#multilevel_duration_function",
+			"label": "Pushbutton 3 Single-Activation Duration Value"
+		},
+		"15[0x01]": {
+			"$import": "templates/logic_group_template.json#multilevel_enable",
+			"label": "Pushbutton 3 Double-Activation Enable"
+		},
+		"15[0xff00]": {
+			"$import": "templates/logic_group_template.json#multilevel_upper_function",
+			"label": "Pushbutton 3 Double-Activation ON Value"
+		},
+		"15[0xff0000]": {
+			"$import": "templates/logic_group_template.json#multilevel_lower_function",
+			"label": "Pushbutton 3 Double-Activation OFF Value"
+		},
+		"15[0xff000000]": {
+			"$import": "templates/logic_group_template.json#multilevel_duration_function",
+			"label": "Pushbutton 3 Double-Activation Duration Value"
+		},
+		"17[0x01]": {
+			"$import": "templates/logic_group_template.json#multilevel_enable",
+			"label": "Pushbutton 4 Single-Activation Enable"
+		},
+		"17[0xff00]": {
+			"$import": "templates/logic_group_template.json#multilevel_upper_function",
+			"label": "Pushbutton 4 Single-Activation ON Value"
+		},
+		"17[0xff0000]": {
+			"$import": "templates/logic_group_template.json#multilevel_lower_function",
+			"label": "Pushbutton 4 Single-Activation ON Value"
+		},
+		"17[0xff000000]": {
+			"$import": "templates/logic_group_template.json#multilevel_duration_function",
+			"label": "Pushbutton 4 Single-Activation OFF Value"
+		},
+		"18[0x01]": {
+			"$import": "templates/logic_group_template.json#multilevel_enable",
+			"label": "Pushbutton 4 Double-Activation Enable"
+		},
+		"18[0xff00]": {
+			"$import": "templates/logic_group_template.json#multilevel_upper_function",
+			"label": "Pushbutton 4 Double-Activation ON Value"
+		},
+		"18[0xff0000]": {
+			"$import": "templates/logic_group_template.json#multilevel_lower_function",
+			"label": "Pushbutton 4 Double-Activation OFF Value"
+		},
+		"18[0xff000000]": {
+			"$import": "templates/logic_group_template.json#multilevel_duration_function",
+			"label": "Pushbutton 4 Double-Activation Duration Value"
+		}
+	},
+	"metadata": {
+		"inclusion": "Remove white pushbutton cover and press on the center switch with a non-conductive object. The LEDs will now start blinking on pushbutton 1 (upper left pushbutton)",
+		"exclusion": "Remove white pushbutton cover and press on the center switch with a non-conductive object. The LEDs will now start blinking on pushbutton 1 (upper left pushbutton)",
+		"reset": "Remove white pushbutton cover and long-press the center switch for 10 seconds with a non-conductive object. Please use this procedure only when the network primary controller is missing or otherwise inoperable",
+		"manual": ""
```

Comment:
Don't add empty keys please

Reply:
What link should i add for the manual if the manual is not located in the ZWA database ?

Reply:
Nothing, just don't add that line.

Reply:
Understood, and removed for future push

Reply:
resolved